### DISCUSSION TITLE
UX : Editing "Entries" fields

### DISCUSSION THREAD

**Discussion by fab4real** [craftcms/cms#8738] [Ideas]
### Description

We use a lot "Entries" type of fields.

When editing a content using this type field, it's possible to edit the related Entry directly (after double clicking on it, a popup appears)

We noticed many end users did not realized the Related Entry is edited globally using this popup. They think it will be edited only for the content thay are editing.

There is many use cases where end users will think they can edit entries from this popup, and will not think about the impact on other places where it appears..

Would it be possible to disable this "edit popup" directly from where Related Entries are used ?

**Comment by brandonkelly** [comment]
Interesting. I don’t think we will make it possible to disable it, but we will think of ways to make it more clear that they are editing the actual entry, not relation-specific content, in Craft 4.

**Comment by fab4real** [comment]
Cool, thanks
